exercises/secret-handshake/README.md

```diff
@@ -39,7 +39,7 @@ please see the [help page](http://exercism.io/languages/python).
 
 ## Source
 
-Bert, in Mary Poppins [http://www.imdb.com/character/ch0011238/quotes](http://www.imdb.com/character/ch0011238/quotes)
+Bert, in Mary Poppins [http://www.imdb.com/title/tt0058331/quotes/qt0437047](http://www.imdb.com/title/tt0058331/quotes/qt0437047)
```

Comment:
It's redundant to use link syntax here, since the text itself is a link.
In other words:
```markdown
http://www.imdb.com/title/tt0058331/quotes/qt0437047
```
would be better than
```markdown
[http://www.imdb.com/title/tt0058331/quotes/qt0437047](http://www.imdb.com/title/tt0058331/quotes/qt0437047)
```